### PR TITLE
Refactored test cases

### PR DESCRIPTION
**Pull request by Schlvf**
## Changes:
- Refactored Logout test case
- Refactored Recycle bin test case
- Refactored account view test case
- Implemented hover and current view title validation common steps

### CODE REVIEW

**Review by svaleria — APPROVED** [review]

**Review by gioacchinogioffre — APPROVED** [review]

File: 2.Views/WebAppPages/HomePage.cs [hunk 1]
```diff
@@ -243,13 +243,13 @@ public class HomePage
...
-        new Locator(
-            LocatorType.XPath,
-            $"//div[@id='ProjectListPlaceHolder']//td[text()='{projectName}']"
-        )
-    );
+        new Button(
+            "",
+            new Locator(
+                LocatorType.XPath,
+                $"//div[@id='ProjectListPlaceHolder']//td[text()='{projectName}']"
```
Comment (sebasonti-jalauni) [inline 2.Views/WebAppPages/HomePage.cs:250]: Please follow the instructions in the comments above
```
// DON'T USE THIS [DEPRECATED]
// USE ProjectButton INSTEAD
```
Also, remove this method if unused.

File: 2.Views/WebAppPages/HomePage.cs [hunk 2]
```diff
@@ -329,5 +329,8 @@ public class HomePage
         new Button("", new Locator(LocatorType.XPath, $"//div[@class='NoItems']"));
 
     public IElement GetItemByIndex(int index) =>
```
Comment (sebasonti-jalauni) [inline 2.Views/WebAppPages/HomePage.cs:331]: Change this and any other method into an element with a dynamic locator.

File: 3.Tests/UI/Features/RecycleBin/DeleteAll.feature [hunk 3]
```diff
@@ -5,7 +5,9 @@ Feature: Empty recycle bin
 
     @Regression
     Scenario: Empty the recycle bin succesfully
-        When the user clicks the recycle bin context menu and clicks the empty recycle bin button
+        When the user hovers on 'Recycle Bin Div' on 'Home Page'
+        And clicks on 'Recycle Bin Dropdown'
+        And clicks on 'Empty Recycle Bin'
         Then the main title text is "Recycle Bin"
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/Features/RecycleBin/DeleteAll.feature:11]: Is this a common step? if not please create one.

File: 3.Tests/UI/Features/Settings/Account/View.feature [hunk 4]
```diff
@@ -4,6 +4,7 @@ Feature: Settings account view
         Given the user is logged in
 
     @Regression
-    Scenario: Empty the recycle bin succesfully
-        When the user clicks on the Settings option and then account tab
+    Scenario: View account settings succesfully
+        When the user clicks on 'Settings' on 'Home Page'
+        And clicks on 'Account'
         Then the user should be able to see the delete account button
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/Features/Settings/Account/View.feature:10]: Use the common verification step for this Then step.

File: 3.Tests/UI/StepDefinitions/RecycleBin/DeleteAllStepDefinition.cs [hunk 5]
```diff
@@ -21,25 +21,6 @@ public DeleteAllStepDefinitions(ScenarioContext scenarioContext) : base(scenario
...
-    {
-        GenericWebDriver.Wait.Until(
-            ExpectedConditions.TextToBePresentInElement(
-                _homePage.ProjectTitleDiv.WebElement,
-                expectedTitle
-            )
-        );
-    }
-
     [Then(@"the snack bar message is ""(.*)""")]
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/StepDefinitions/RecycleBin/DeleteAllStepDefinition.cs:24]: This step can be moved to the CommonSteps class.

**Review by sebasonti-jalauni — CHANGES_REQUESTED** [review]
A few refactoring tasks missing

**Pull request merged**